fixham/internal/version/version.go

```diff
@@ -0,0 +1,79 @@
+package version
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Masterminds/semver/v3"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+
+	"github.com/projectcalico/calico/fixham/internal/calico"
+)
+
+type Version string
+
+func (v *Version) Set(value string) error {
+	_v, err := semver.NewVersion(strings.TrimPrefix(value, "v"))
```

Comment:
`_v` stands out a bit in Go - it's not a very command way of defining a variable (unlike Python).

I'd call this `ver` probably